<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>191</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Restaurants:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>200</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Restaurant Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>471</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Menu Items: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="listView"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Distance (miles)
to other Fast Food restaurants</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="listView_2"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>310</y>
      <width>275</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Miles to Saddleback College:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="actionUpdate_List"/>
   </widget>
   <widget class="QMenu" name="menuAdmin">
    <property name="title">
     <string>Admin</string>
    </property>
    <addaction name="actionLog_in"/>
    <addaction name="actionLog_out"/>
   </widget>
   <widget class="QMenu" name="menuTrips">
    <property name="title">
     <string>Trips</string>
    </property>
    <addaction name="action10_Closest"/>
    <addaction name="actionShortest_Trip"/>
    <addaction name="actionVisit_All_12"/>
    <addaction name="actionCustom_Trip"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAdmin"/>
   <addaction name="menuTrips"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="toolTip">
    <string>Import the Initial Restaurants</string>
   </property>
  </action>
  <action name="actionUpdate_List">
   <property name="text">
    <string>Update List</string>
   </property>
   <property name="toolTip">
    <string>Add the Extra 2 Restaurants</string>
   </property>
  </action>
  <action name="actionLog_in">
   <property name="text">
    <string>Log-in</string>
   </property>
   <property name="toolTip">
    <string>Log-in for Maintinance</string>
   </property>
  </action>
  <action name="actionLog_out">
   <property name="text">
    <string>Log-out</string>
   </property>
   <property name="toolTip">
    <string>Log-out of Maintinance</string>
   </property>
  </action>
  <action name="action10_Closest">
   <property name="text">
    <string>10 Closest</string>
   </property>
   <property name="toolTip">
    <string>Plan a Trip to the Closest 10 Restaurants</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
  </action>
  <action name="actionShortest_Trip">
   <property name="text">
    <string>Shortest Trip</string>
   </property>
   <property name="toolTip">
    <string>Plan the Shortest Trip</string>
   </property>
  </action>
  <action name="actionVisit_All_12">
   <property name="text">
    <string>Visit All 12</string>
   </property>
   <property name="toolTip">
    <string>Visit All 12 Restaurants</string>
   </property>
  </action>
  <action name="actionCustom_Trip">
   <property name="text">
    <string>Custom Trip</string>
   </property>
   <property name="toolTip">
    <string>Customize a Trip to Take</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
